--- apache-tomcat-7.0.54/bin/service.bat	2014-05-19 22:33:02.000000000 -0400
+++ ozone-tomcat-7.0.54/bin/service.bat	2014-07-25 22:23:27.108429763 -0400
@@ -142,6 +142,9 @@
 set "CLASSPATH=%CATALINA_HOME%\bin\bootstrap.jar;%CATALINA_BASE%\bin\tomcat-juli.jar"
 if not "%CATALINA_HOME%" == "%CATALINA_BASE%" set "CLASSPATH=%CLASSPATH%;%CATALINA_HOME%\bin\tomcat-juli.jar"
 
+rem Add OZONE-specific Java options
+set OZ_J_OPTS=-Djavax.net.ssl.trustStore=%CATALINA_HOME%\certs\keystore.jks;-Djavax.net.ssl.keyStore=%CATALINA_HOME%\certs\keystore.jks;-Djavax.net.ssl.trustStorePassword=changeit;-Djavax.net.ssl.keyStorePassword=changeit;-XX:PermSize=128m;-XX:MaxPermSize=256m
+
 "%EXECUTABLE%" //IS//%SERVICE_NAME% ^
     --Description "Apache Tomcat 7.0.54 Server - http://tomcat.apache.org/" ^
     --DisplayName "%DISPLAYNAME%" ^
@@ -159,7 +162,7 @@
     --StopClass org.apache.catalina.startup.Bootstrap ^
     --StartParams start ^
     --StopParams stop ^
-    --JvmOptions "-Dcatalina.home=%CATALINA_HOME%;-Dcatalina.base=%CATALINA_BASE%;-Djava.endorsed.dirs=%CATALINA_HOME%\endorsed;-Djava.io.tmpdir=%CATALINA_BASE%\temp;-Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager;-Djava.util.logging.config.file=%CATALINA_BASE%\conf\logging.properties" ^
+    --JvmOptions "%OZ_J_OPTS% -Dcatalina.home=%CATALINA_HOME%;-Dcatalina.base=%CATALINA_BASE%;-Djava.endorsed.dirs=%CATALINA_HOME%\endorsed;-Djava.io.tmpdir=%CATALINA_BASE%\temp;-Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager;-Djava.util.logging.config.file=%CATALINA_BASE%\conf\logging.properties" ^
     --JvmMs 128 ^
     --JvmMx 256
 if not errorlevel 1 goto installed
